### DISCUSSION TITLE
Error using "go get github.com/jptosso/coraza-waf/v2@latest"

### DISCUSSION THREAD

**Discussion by briandoesdev** [corazawaf/coraza#347] [Q&A] [answered]
Okay, I don't work in Go lang all that often (ever really). So this may just be a mess up on my end. But following the instructions [here](https://coraza.io/docs/tutorials/quick-start/) I get the following error on the first step "Add Coraza to your go project":

```
go: go.mod file not found in current directory or any parent directory.
        'go get' is no longer supported outside a module.
        To build and install a command, use 'go install' with a version,
        like 'go install example.com/cmd@latest'
        For more information, see https://golang.org/doc/go-get-install-deprecation
        or run 'go help get' or 'go help install'.
```

I'm running the following version of Go: `go version go1.19 darwin/amd64`

Am I just missing some beginner Go knowledge that would fix this?

**Comment by syinwu** [comment, marked as answer]
Hey! It looks like you need to execute ```go get``` in the go mod directory. Example:
```
mkdir example && cd example
go mod init example.com
go get ...
```
About go mod: [https://go.dev/ref/mod](https://go.dev/ref/mod)

**Reply by briandoesdev** [reply]
I thought it'd be something simple.. Looks like I should watch a couple videos on Go basics so that doesn't happen again.

Thank you!

**Reply by jptosso** [reply]
There is an example you can use: https://github.com/corazawaf/coraza/tree/v2/master/examples/http-server

Just copy the directory, then run `go mod init` inside of it and start building
